mycroft/skills/common_iot_skill.py

```diff
@@ -0,0 +1,340 @@
+# Copyright 2018 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# THE CLASSES IN THIS FILE ARE STILL EXPERIMENTAL, AND ARE SUBJECT TO
+# CHANGES. IT IS PROVIDED NOW AS A PREVIEW, SO SKILL AUTHORS CAN GET
+# AN IDEA OF WHAT IS TO COME. YOU ARE FREE TO BEGIN EXPERIMENTING, BUT
+# BE WARNED THAT THE CLASSES, FUNCTIONS, ETC MAY CHANGE WITHOUT WARNING.
+
+from abc import ABC, abstractmethod
+from enum import Enum
+from mycroft import MycroftSkill
+from mycroft.messagebus.message import Message
+from mycroft.util.log import getLogger
+
+
+LOGGER = getLogger(__name__)
```

Comment:
We usually do 

```python
from mycroft.util.log import LOG
```
and then just use

```python
LOG.info('This is useful information')
```

Reply:
Removed this. The logger was only being used for some debugging yesterday, and I just forgot to remove it.